modules/installation-aws-arm-tested-machine-types.adoc

```diff
@@ -10,16 +10,16 @@
 // installing/installing_aws/installing-restricted-networks-aws.adoc
 
 [id="installation-aws-arm-tested-machine-types_{context}"]
-= Tested instance types for AWS ARM
+= Tested instance types for AWS on 64-bit ARM infrastructures
 
-The following Amazon Web Services (AWS) ARM instance types have been tested with {product-title}.
+The following Amazon Web Services (AWS) ARM64 instance types have been tested with {product-title}.
```

Comment:
Is `ARM64` how AWS refers to these instance types. This is probably me not just understanding the guidelines you are applying, but I am seeing `64-bit ARM instances` in `installing/installing_preparing.adoc`

Reply:
AWS uses a few formats, I've seen 64-bit Arm, ARM64, and arm64. But I think I'll change it to 64-bit ARM to be more general.